tests/execution_engine/split_and_sample/test_sqlalchemy_execution_engine_splitting.py

```diff
@@ -315,18 +315,29 @@ def test_get_split_query_for_data_for_batch_identifiers_for_split_on_date_parts_
     )
 
 
+@pytest.mark.integration
 @pytest.mark.parametrize(
     "date_parts",
     MULTIPLE_DATE_PART_DATE_PARTS,
 )
+@pytest.mark.parametrize(
+    "dialect,expected_query_str",
+    [
+        pytest.param(dialect, expected_query_str, id=dialect)
+        for dialect, expected_query_str in {
+            "sqlite": "SELECT DISTINCT(CAST(EXTRACT(year FROM column_name) AS VARCHAR) || CAST (EXTRACT(month FROM column_name) AS VARCHAR)) AS concat_distinct_values, CAST(EXTRACT(year FROM column_name) AS INTEGER) AS year, CAST(EXTRACT(month FROM column_name) AS INTEGER) AS month FROM table_name",
```

Comment:
@Shinnnyshinshin Yes, I would expect most backends to use different queries. It's a tough call regarding how many we should be doing integration tests against when we haven't really setup a framework for testing against most backends. What would the parametrization you are describing look like? I already looked for ways to do this, but this is what I found.

Reply:
@NathanFarmer  

1. Ok that sounds totally reasonable. Maybe as we harden the integration to various backends as part of Luigi-pod work, this will be more fleshed-out
2. Regarding the example I had in mind, could you have a look at the tests here?
 `https://github.com/great-expectations/great_expectations/blob/8f0eb085b31ab81f073ff8b4606cf5fd42eb047e/tests/core/test_serialization.py#L1055`

Reply:
✅